scripts/old/exgdas_global_marine_analysis_vrfy.py

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/env python3
```

Comment:
what is that file for?

Reply:
Removed-

Reply:
... Not sure where my previous comment went ... I still see the file @apchoiCMD

Reply:
Your previous comment was clear but do we need to keep this script in the `old` folder when we reverse `vrfy` into the workflow again? If you don't mind, I want to keep it in the `old` directory.

Reply:
Removed-